benchmark/utils/formats.hpp

```diff
@@ -53,7 +53,11 @@ namespace formats {
 
 
 std::string available_format =
-    "coo, csr, ell, ell_mixed, sellp, hybrid, hybrid0, hybrid25, hybrid33, "
+    "bccoo, bccoo-elm, bccoo-elm-32, bccoo-elm-64, bccoo-elm-128, "
+    "bccoo-elm-256, bccoo-elm-512, bccoo-elm-1024, "
+    "bccoo-blk, bccoo-blk-32, bccoo-blk-64, bccoo-blk-128, "
+    "bccoo-blk-256, bccoo-blk-512, bccoo-blk-1024, "
+    "coo, csr, ell, ell-mixed, sellp, hybrid, hybrid0, hybrid25, hybrid33, "
```

Comment:
```suggestion
    "coo, csr, ell, ell_mixed, sellp, hybrid, hybrid0, hybrid25, hybrid33, "
```
the option is ell_mixed

Reply:
Should I change "-" to "_" in blk options?

Reply:
I will use "_"

Reply:
Done!!